recipes/opentelemetry-cpp/all/conanfile.py

```diff
@@ -64,6 +64,7 @@ def requirements(self):
         self.requires("thrift/0.17.0")
         if Version(self.version) >= "1.3.0":
             self.requires("boost/1.80.0")
+        self.requires("zlib/1.2.13")
```

Comment:
The zlib requirement is no longer necessary.
I couldn't build opentelemetry-cpp because of the zlib dependency issue of boost and thrift.
The recipes for thrift and boost have been updated, so this line is no longer needed.
```suggestion
```
https://github.com/conan-io/conan-center-index/pull/13884
https://github.com/conan-io/conan-center-index/pull/13909

Reply:
It is trivial.
I will leave it up to you to correct it.

Reply:
Thanks, I have removed.